### DISCUSSION TITLE
cn/2020/01/21/hugo-%E5%A6%82%E4%BD%95%E5%AE%9E%E7%8E%B0%E8%84%9A%E6%B3%A8/

### DISCUSSION THREAD

**Discussion by giscus** [hongtaoh/hongtaoh.github.io#169] [Announcements]
# cn/2020/01/21/hugo-%E5%A6%82%E4%BD%95%E5%AE%9E%E7%8E%B0%E8%84%9A%E6%B3%A8/

https://hongtaoh.com/cn/2020/01/21/hugo-%E5%A6%82%E4%BD%95%E5%AE%9E%E7%8E%B0%E8%84%9A%E6%B3%A8/

**Comment by daoyuanw** [comment]
改得不方便了，容易把脚注后的正文也给标记了。

**Reply by hongtaoh** [reply]
是的
